### ISSUE TITLE
Duplicate Y-Axis Labels

### ISSUE THREAD

**Issue by bwjohns4** [indiespirit/react-native-chart-kit#613]
Has anyone encountered this issue where there are duplicate Y axis labels?

<img width="482" alt="image" src="https://user-images.githubusercontent.com/29932360/173696183-57c39def-ee43-4728-b9da-defa5b37f0fb.png">

**Comment by moisfelix** [comment]
Yes, me. Basically this are not duplicated, if you set decimals to 1 or 2 you will see in fact are different values. But if y values are in fact 1,7 and 2.4 both values are rounded at 2.

**Comment by bwjohns4** [comment]
Interesting. My application only uses whole numbers. Is there a way to set this up so that it does not show y-axis increments of less than a whole number increment?

I'm already using: yAxisInterval={1}

**Comment by BuniEmris** [comment]
@bwjohns4  I also faced issue. the problem was as there are 5 lines of  Y axis,all given data value will fill these lines  my initial value is set to zero : fromZero={true}  , 
if max value is less than 4 , the other values will be duplicated to fill 5 lines,
 so the solution is you need also give max value like this:
 fromNumber={Math.max(...arrdata) > 4 ? Math.max(...arrdata) : 4 }  
<img width="180" alt="Screen Shot 2022-07-24 at 20 44 05" src="https://user-images.githubusercontent.com/66415679/180655303-729c25d0-a68b-4840-a2da-5d283c9874bc.png">
 if data value is higher than 4 then it draw like this:
<img width="135" alt="Screen Shot 2022-07-24 at 20 44 59" src="https://user-images.githubusercontent.com/66415679/180655336-57680648-be09-465e-b17f-0fc295dd784e.png">
<img width="690" alt="Screen Shot 2022-07-24 at 20 48 55" src="https://user-images.githubusercontent.com/66415679/180655368-acaee6a8-cb78-43ce-a3b7-72262987d838.png">

**Comment by BuniEmris** [comment]
<img width="180" alt="Screen Shot 2022-07-24 at 20 44 05" src="https://user-images.githubusercontent.com/66415679/180655169-74b0a606-1f02-4709-9ded-870ac571c46a.png">

**Comment by BuniEmris** [comment]
if value is higher than 4 then it draw like this
<img width="135" alt="Screen Shot 2022-07-24 at 20 44 59" src="https://user-images.githubusercontent.com/66415679/180655212-5ecffb3c-1657-4dad-a368-7e8e93230a65.png">

**Comment by Girishav** [comment]
@BuniEmris Thank you for above solution. I had similar issue and suggesstions worked like a charm.

**Comment by ZainAliSyed24** [comment]
> @bwjohns4 I also faced issue. the problem was as there are 5 lines of Y axis,all given data value will fill these lines my initial value is set to zero : fromZero={true} , if max value is less than 4 , the other values will be duplicated to fill 5 lines, so the solution is you need also give max value like this: fromNumber={Math.max(...arrdata) > 4 ? Math.max(...arrdata) : 4 } <img alt="Screen Shot 2022-07-24 at 20 44 05" width="180" src="https://user-images.githubusercontent.com/66415679/180655303-729c25d0-a68b-4840-a2da-5d283c9874bc.png"> if data value is higher than 4 then it draw like this: <img alt="Screen Shot 2022-07-24 at 20 44 59" width="135" src="https://user-images.githubusercontent.com/66415679/180655336-57680648-be09-465e-b17f-0fc295dd784e.png"> <img alt="Screen Shot 2022-07-24 at 20 48 55" width="690" src="https://user-images.githubusercontent.com/66415679/180655368-acaee6a8-cb78-43ce-a3b7-72262987d838.png">

Thanks @BuniEmris  It works for me

**Comment by TrevorGarnett** [comment]
I am having the same issue for a LineChart. If I allow it to show more decimal places, it shows {1,1.5,2,2.5,3}. Is there a similar solution for a LineChart? LineCharts unfortuantely don' thave "showBarTops" property.

![IMG_366C560338BA-1](https://github.com/indiespirit/react-native-chart-kit/assets/54717962/a9278acf-1bf0-4312-85f1-0b4c4bbb7a41)
